Aikido.Zen.Core/Helpers/IPHelper.cs

```diff
@@ -7,6 +7,62 @@ namespace Aikido.Zen.Core.Helpers
 {
     public class IPHelper
     {
+        private static readonly IPRange _privateIpRanges = InitializePrivateRanges();
+
+        private static IPRange InitializePrivateRanges()
+        {
+            var ranges = new IPRange();
+
+            // IPv4 private ranges - updated to include all reserved ranges
+            // taken from https://github.com/frenchbread/private-ip/blob/master/src/index.ts
```

Comment:
Would just point to node repo here, since this list should be the same across firewalls?